massa-grpc/src/api.rs

```diff
@@ -339,6 +339,29 @@ pub(crate) fn get_largest_stakers(
     })
 }
 
+// Get node version
+pub(crate) fn get_mip_status(
+    grpc: &MassaGrpc,
+    request: tonic::Request<grpc::GetMipStatusRequest>,
+) -> Result<grpc::GetMipStatusResponse, GrpcError> {
+    let mip_store_status_ = grpc.mip_store.get_mip_status();
+    let mip_store_status: Result<Vec<grpc::MipStatusEntry>, GrpcError> = mip_store_status_
+        .iter()
+        .map(|(mip_info, state_id_)| {
+            let state_id = grpc::ComponentStateId::from(state_id_);
+            Ok(grpc::MipStatusEntry {
+                mip_info: Some(grpc::MipInfo::from(mip_info)),
+                state_id: i32::from(state_id),
```

Comment:
This seems to be buggy, Rust enum starts at 0 and gRPC one starts at 1. 

To avoid this kind of issue, creating a Into trait will make It explicit

Reply:
Ah sorry, It's good